libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -217,16 +229,15 @@ const ParentalLeaveTemplate: ApplicationTemplate<
                 import('../forms/InReview').then((val) =>
                   Promise.resolve(val.InReview),
                 ),
-              read: {
-                answers: ['spread', 'periods'],
-                externalData: ['pregnancyStatus', 'parentalLeaves'],
-              },
+              read: 'all',
+              write: 'all',
```

Comment:
I'm still unfamiliar with the state machine, but does it mean that the employer can write on all his own answers or write on the applicant answers?

Reply:
In this case it would give the applicant (or whatever role defined in that scope) the rights to read and write all answers in the application.

Reply:
It should work as the employer only being able to read a certain part, and submit or reject. 
On line 216 we set the permission for the `Roles.ASSIGNEE` which would be the employer: 
`read: { answers: ['periods'], externalData: ['pregnancyStatus'] },` 

Then for `Roles.APPLICANT` we allow them to write/submit (so that they can set it back to 'draft').

@baering is that correct?